web/src/components/Terminal.tsx

```diff
@@ -0,0 +1,480 @@
+import clsx from 'clsx'
+import { useEffect, useRef, useState } from 'react'
+
+export default () => {
+    return (
+        <div className='w-full flex flex-col bg-gray-900 rounded-2xl overflow-hidden relative'>
+            <Header />
+            <Content />
+        </div>
+    )
+}
+
+const Header = () => {
+    const colors = [
+        'bg-red-500',
+        'bg-yellow-500',
+        'bg-green-500',
+    ]
+    return (
+        <div className='flex w-full border-b border-b-slate-800 bg-gray-900 '>
+            {colors.map(color => (
+                <div
+                    className={clsx(
+                        'h-3 w-3 ml-2 my-2 rounded-full',
+                        color
+                    )}
+                    key={color}
```

Comment:
I've found it safer for the key to be the index of the iteration instead of the color itself.

Reply:
In this case, the colors will always be unique. I'll take note of that though.